modules/directives/src/main/scala/scala/build/preprocessing/directives/UsingPublishDirectiveHandler.scala

```diff
@@ -50,54 +48,62 @@ case object UsingPublishDirectiveHandler extends UsingDirectiveHandler {
     "gpg-options"
   ).map(prefix + _)
 
+  override def getValueNumberBounds(key: String) = UsingDirectiveValueNumberBounds(1, 1)
+
   def handleValues(
-    directive: StrictDirective,
-    path: Either[String, os.Path],
-    cwd: ScopePath,
+    scopedDirective: ScopedDirective,
     logger: Logger
-  ): Either[BuildException, ProcessedUsingDirective] = either {
-    def singleValue   = DirectiveUtil.singleStringValue(directive, path, cwd)
-    def severalValues = DirectiveUtil.stringValues(directive.values, path, cwd)
+  ): Either[BuildException, ProcessedUsingDirective] =
+    checkIfValuesAreExpected(scopedDirective).flatMap { groupedScopedValuesContainer =>
+      // This head is fishy!
```

Comment:
Sort of since we specify that we will have at least single element

Reply:
would definitely double check that...